vtex-apps/order-placed
# Remove unnecessary context.json and update app version

#### What is the purpose of this pull request?


Re-publishes the `unit` translation fix and remove the unnecessary `context.json` file.

#### What problem is this solving?

Re-publishes the `unit` translation fix and remove the unnecessary `context.json` file.

#### How should this be manually tested?

Log into https://kiwi--alssports.myvtex.com/checkout/orderPlaced/?og=943191334084 and see that it renders `unit`, not `unity`.

#### Screenshots or example usage

Before:
![image](https://user-images.githubusercontent.com/12702016/61469554-c8f00c00-a955-11e9-9c40-bef1efe89510.png)

After:
![image](https://user-images.githubusercontent.com/12702016/61469531-bc6bb380-a955-11e9-88d1-730268912384.png)


#### Types of changes
